README.md

```
@@ -1,14 +1,31 @@
+- [Flutter と Firebase を使ってアプリを作る上でのテクニックあれこれ](https://zenn.dev/kingu/articles/f122c06a9cc360)
+
+## ダウンロード
+
+### iOS 版
+
+<a href="https://apps.apple.com/jp/app/blooms/id6740260241?itscg=30200&itsct=apps_box_badge&mttnsubad=6740260241" style="display: inline-block;">
+<img src="https://toolbox.marketingtools.apple.com/api/v2/badges/download-on-the-app-store/black/ja-jp?releaseDate=1738886400" alt="App Storeでダウンロード" style="width: 224px; height: 82px; vertical-align: middle; object-fit: contain;" />
+</a>
 
-<https://apps.apple.com/jp/app/blooms/id6740260241>
+<img src="assets/qr-code.webp" width="256" height="256" alt="QR Code" />
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: img

(MD033, no-inline-html)